rfcs/text/0042/risk.yml

```diff
@@ -97,7 +97,7 @@
       type: float
       example: 75.0
       description: >
-        The contribution of Category 5 to the overall risk score (`calculated_score`).
+        The contribution of Category 5 to the overall normalized risk score (`calculated_score_norm`).
```

Comment:
@ebeahan @SourinPaul this was the main change I made to convey that these category scores are themselves normalized. I originally had included the phrase "normalized contribution" to be more explicit, but that seemed redundant since the _contribution_ to a normalized score only really makes sense if they can be compared (/are normalized / exist in the same value range, etc). Let me know if you opinions/suggestions.